### DISCUSSION TITLE
Statistics: `Articles repartition` should specify date or else it is misleading

### DISCUSSION THREAD

**Discussion by lamyergeier** [FreshRSS/FreshRSS#6903] [Ideas]
# Reference Image

- For feed URL: <https://0pointer.net/blog/index.rss20>

![image](https://github.com/user-attachments/assets/de483e73-270b-4b1f-83d4-19e0c6ffdf76)

# Issue

For a particular feed, it seems that it has been posting on an average 1.74 messages. But this is misleading as the following image shows that there are only 2 entries in this year of 2024:

![image](https://github.com/user-attachments/assets/58ad706b-ef7a-48a5-9afb-e462495cddb1)

# Solution

Let user enter the date from which the statistics are computed!

**Comment by math-GH** [comment]
lets discuss it as issue here: https://github.com/FreshRSS/FreshRSS/issues/6905
